test/integration/data-types.test.js

```diff
@@ -378,12 +378,31 @@ describe(Support.getTestDialectTeaser('DataTypes'), () => {
 
     it('should parse an empty GEOMETRY field', () => {
       const Type = new Sequelize.GEOMETRY();
-
       // MySQL 5.7 or above doesn't support POINT EMPTY
       if (dialect === 'mysql' && semver.gte(current.options.databaseVersion, '5.7.0')) {
         return;
       }
 
+      it('should return parenteses when default values is a function', function() {
```

Comment:
Some changes

1. This `it` block is placed inside another `it` block so it won't execute. Please move it outside of `    it('should parse an empty GEOMETRY field', () => {` block
2. You should use async/await